### DISCUSSION TITLE
How to install Symfony 5

### DISCUSSION THREAD

**Discussion by axelleP** [symfony/symfony#42779] [Q&A] [answered]
Hello, i want to install Symfony 5, so i followed the installation of https://symfony.com/download but i get Symfony 4.4.29 instead of 5. I don't understand.
Thanks for your help.

**Comment by ahundiak** [comment, marked as answer]
PHP version? Needs to be at least 7.2.5 for Symfony 5 to install.  Probably should double check the Symfony CLI executable version as well.  The latest version is currently at 4.25.6.

**Reply by axelleP** [reply]
Yes, in fact my WAMPSERVER is on PHP 7.4.11 but my PATH variable PHP was on 7.2.4. so Symfony runned with 7.2.4...
I changed that, now it's work, thanks for your help.
